SU-HSDO/suhumsci
# shs-2: fix alignment of media asset edit button in wysiwyg

# READY FOR REVIEW

## Summary
This fixes a bug introduced by CK Editor styles involving the 'Edit' button on media in the WYSIWYG editor.

## Urgency
_'low'_

## Screenshot
![Screen Shot 2022-05-04 at 1 40 12 PM](https://user-images.githubusercontent.com/8405274/166747279-f80ba862-c20b-463c-a86d-534180c8aaa8.png)

## Steps to Test
1. Pull down the branch, navigate to the `humsci_basic` theme folder, and run `npm run watch` after installing the required modules.
2. In Drupal, go to a 'Flexible Page'. Add an image via the WYSIWYG. Hover over the image and verify the edit button is centered instead of off-centered and weird.

## Questions
- The `watch`/`build` command outputs changes for Airy/Traditional/Colorful. Should these be included or is that handled by a build process for the appropriate theme?
- The solution to this bug could have also been handled by modifying the custom module styles that controls the edit button. Is that approach ever taken? If so, can we get access to those repos?
- Are labels heavily-used here or optional?
